config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -258,6 +255,49 @@ presubmits:
           requests:
             cpu: "7"
             memory: 9000Mi
+  # for testing new configurations *without* blocking merges*
+  # NOTE: changes should generally be synced to pull-kubernetes-e2e-kind-canary
+  - name: pull-kind-e2e-kubernetes-canary
+    cluster: k8s-infra-prow-build
+    always_run: false
+    optional: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    path_alias: sigs.k8s.io/kind
+    decoration_config:
+      timeout: 60m
+      grace_period: 15m
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:latest-master
+        imagePullPolicy: Always # pull latest image for canary testing
+        command:
+        - wrapper.sh
+        - bash
+        - -c
+        - cd ./../../k8s.io/kubernetes && ./../../sigs.k8s.io/kind/hack/ci/e2e.sh
+        env:
+        - name: LABEL_FILTER
+          value: "Feature: isSubsetOf OffByDefault && !Slow && !Disruptive && !Flaky"
```

Comment:
Oh, I'm not sure why I was thinking this, we aren't enabling offbydefault and shouldn't in this job. So Feature: isEmpty makes more sense

Reply:
https://github.com/kubernetes/test-infra/pull/34600